mon-pix/tests/integration/components/authentication/oidc-reconciliation_test.js

```diff
@@ -0,0 +1,54 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import Service from '@ember/service';
+import { render } from '@1024pix/ember-testing-library';
+import { hbs } from 'ember-cli-htmlbars';
+import setupIntlRenderingTest from '../../../helpers/setup-intl-rendering';
+
+describe('Integration | Component |  authentication | oidc-reconciliation', function () {
+  setupIntlRenderingTest();
+
+  it('should display reconciliation page elements', async function () {
+    // given
+    const oidcPartner = {
+      organizationName: 'Partenaire OIDC',
+    };
+    class OidcIdentityProvidersStub extends Service {
+      'oidc-partner' = oidcPartner;
+      list = [oidcPartner];
+    }
+    this.owner.register('service:oidcIdentityProviders', OidcIdentityProvidersStub);
+    this.set('fullNameFromPix', 'Lloyd Pix');
+    this.set('fullNameFromExternalIdentityProvider', 'Lloyd Cé');
+    this.set('email', 'lloyidce@example.net');
+    this.set('identityProviderSlug', 'oidc-partner');
+    this.set('authenticationMethods', [{ identityProvider: 'CNAV' }]);
```

Comment:
Suggestion : si la possibilité d'avoir une methode en plus existe, ça serait cool de le voir apparaître ici (un tableau d'objets me laisse supposer qu'il peut y en avoir plus d'une) 
`this.set('authenticationMethods', [{ identityProvider: 'CNAV' }, { identityProvider: 'POLE_EMPLOI' } ]);`

Reply:
done ✅